synapse/federation/transport/server.py

```diff
@@ -261,10 +261,10 @@ def new_func(request, *args, **kwargs):
             except NoAuthenticationError:
                 origin = None
                 if self.REQUIRE_AUTH:
-                    logger.exception("authenticate_request failed")
+                    logger.warn("authenticate_request failed")
```

Comment:
can you log the value of the exception so we have a clue *why* it failed?

Reply:
`NoAuthenticationError` simply means that the request is missing auth, so I've explicitly said that rather than logging the actual error